.github/workflows/main.yaml

```diff
@@ -0,0 +1,85 @@
+name: Main
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '**'
+    paths-ignore:
+      - README.md
+      - LICENSE
+  pull_request:
+
+jobs:
+  build-image:
+    name: Build image
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - model_name: facebook/bart-large-cnn
+            model_tag_name: facebook-bart-large-cnn
+            onnx_runtime: false
+          - model_name: facebook/bart-large-cnn
+            model_tag_name: facebook-bart-large-cnn
+            onnx_runtime: true
+          - model_name: google/pegasus-xsum
+            model_tag_name: google-pegasus-xsum
+            onnx_runtime: false
+          # - model_name: google/pegasus-xsum
+          #   model_tag_name: google-pegasus-xsum
+          #   onnx_runtime: true
+    env:
+      LOCAL_REPO: sum-transformers
+      REMOTE_REPO: semitechnologies/sum-transformers
+      MODEL_NAME: ${{matrix.model_name}}
+      MODEL_TAG_NAME: ${{matrix.model_tag_name}}
+      ONNX_RUNTIME: ${{matrix.onnx_runtime}}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
```

Comment:
when you update the docker file, change to 3.10/11 here (depending on the base image)

Reply:
I will harden the docker image in the next PR